modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/src/EventSubscriber/WebtoolsAnalyticsEventSubscriber.php

```diff
@@ -50,7 +63,7 @@ public function __construct(EntityTypeManagerInterface $entityTypeManager, Reque
    * @param \Drupal\oe_webtools_analytics\AnalyticsEventInterface $event
    *   Response event.
    */
-  public function setSection(AnalyticsEventInterface $event): void {
+  public function analyticsEventHandler(AnalyticsEventInterface $event): void {
     try {
```

Comment:
We should move the storage retrieval after we check the cache.

Reply:
Fixed. Thanks for note.